commit 234c185e2909622fe6cbc36955f238b261e7dff2
Author: Trent Mick <trentm@gmail.com>
Date:   2019-03-22T09:14:43-07:00 (7 months ago)
    
    TRITON-1345 fix sdc-manta conditional inclusion of agent_prebuilt Makefile so can 'make check' on non-SmartOS

diff --git a/Makefile b/Makefile
index 23ca3ca..aad8213 100644
--- a/Makefile
+++ b/Makefile
@@ -171,8 +171,10 @@ publish: release
 CLEAN_FILES += node_modules
 
 include ./deps/eng/tools/mk/Makefile.deps
-include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
-include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+ifeq ($(shell uname -s),SunOS)
+    include ./deps/eng/tools/mk/Makefile.node_prebuilt.targ
+    include ./deps/eng/tools/mk/Makefile.agent_prebuilt.targ
+endif
 include ./deps/eng/tools/mk/Makefile.targ
 
 MAN_SECTION	:= 1
